src/sentry/api/event_search.py

```diff
@@ -92,12 +99,18 @@
 boolean_filter = negation? search_key sep boolean_value
 
 # aggregate numeric filter
-aggregate_filter          = negation? aggregate_key sep operator? (duration_format / numeric_value / percentage_format)
-aggregate_date_filter     = negation? aggregate_key sep operator? iso_8601_date_format
-aggregate_rel_date_filter = negation? aggregate_key sep operator? rel_date_format
+aggregate_filter = negation? aggregate_key sep operator? (duration_format / numeric_value / percentage_format)
+
+# aggregate for dates
+aggregate_time_filter = negation? aggregate_key sep operator? iso_8601_date_format
```

Comment:
Why the change from date -> time (reads weird when we have `iso_8601_date_format`)

Reply:
since the other ones were called time. But I could switch them all over to date.

Reply:
Okay changed everything to `date`.